docs/general/adapters.md

```diff
@@ -194,6 +194,43 @@ The user could pass in `include=**`.
 
 We recommend filtering any user-supplied includes appropriately.
 
+#### Include Parameters
+
+You may wish to include related resources based on the `include` request
+parameter (See [fetching_includes](http://jsonapi.org/format/#fetching-includes). You can opt-in to this pattern:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  associations_via_include_param(true)
```

Comment:
Good questions! I **absolutely** agree the naming gets confusing here and glad to take any suggestions.

* That you're opting in to this feature. The biggest reason why you might **not** want to opt in, is if you want the `relationships``id/type`